### DISCUSSION TITLE
Mesh Size Effects on Tray Surface Area and HRR Control in Sub-Zero Conditions

### DISCUSSION THREAD

**Discussion by MNJOLE** [firemodels/fds#13527] [General]
Hello, I am currently facing two issues while working with FDS version 6.7.5. I would appreciate your assistance with the following:

Changes in a circular tray due to mesh size: When the mesh size changes, the implementation of the tray is altered, and the surface area of the tray changes. This makes it difficult to analyze the effects of different data based on mesh size. I would like to know how to resolve this issue.

LES evaporation model under sub-zero conditions: I attempted a simple simulation using the LES evaporation model under sub-zero temperature conditions. However, I observed a delay in the ignition start time, and contrary to expectations, the HRR spiked higher than it did under ambient conditions before eventually decreasing. I would like to know how to achieve a lower peak HRR in sub-zero conditions.

Thank you in advance for your help.

**Comment by drjfloyd** [comment]
Assuming your all your grid sizes are still reasonably resolved, you should not be seeing drastic changes in pool area as you refine the grid. Going from something like 8 cells across a round pool to 12 cells is only going to change pool area ~5 %. With quantities like flame height and temperature scaling with a fractional power of the HRR, differences between 8 and 12 cells would largely be from grid effects and not area change.  

If you are trying to predict the burning rate of a liquid pool, this is a research application of FDS. You are using a 4+ year old version of FDS and many changes have been made since 6.7.5 including a number of changes and big fixes to the pyrolysis model in the code (which includes evaporation). First, I suggest you use the current release.
